framework/platform/CMakeLists.txt

```diff
@@ -43,6 +43,17 @@ if (NOT DEFINED TCUTIL_PLATFORM_SRCS)
 		if (NOT (DEQP_SUPPORT_EGL OR DEQP_SUPPORT_GLX))
 		  message(FATAL_ERROR "At least one of EGL and GLX must be enabled for X11")
 		endif ()
+
+	elseif (DEQP_USE_WAYLAND)
+		set(TCUTIL_PLATFORM_SRCS
```

Comment:
In the long run we probably want to get rid of `DEQP_USE_X11` altogether.

`wayland.cmake` can set `TCUTIL_PLATFORM_SRCS` and it will be honored by this file. See for example `targets/raspi/raspi.cmake' for Raspberry Pi support.